README.md

```diff
@@ -0,0 +1,30 @@
+# Connect-4 v1.0
```

Comment:
I'd remove the versioning here. That way it doesn't have to be touched for future versions.